server/plugin/template.go

```diff
@@ -183,13 +183,39 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 {{- end -}}
 `))
 
-	template.Must(masterTemplate.New("newPR").Funcs(funcMap).Parse(`
+	template.Must(masterTemplate.New("newDraftPR").Funcs(funcMap).Parse(`
 {{ if eq .Config.Style "collapsed" -}}
 {{template "repo" .Event.GetRepo}} New pull request {{template "pullRequest" .Event.GetPullRequest}} was opened by {{template "user" .Event.GetSender}}.
 {{- else -}}
 #### {{.Event.GetPullRequest.GetTitle}}
 ##### {{template "eventRepoPullRequest" .Event}}
 #new-pull-request by {{template "user" .Event.GetSender}}
+{{- end }}
+`))
+
+	template.Must(masterTemplate.New("newReadyToReviewPR").Funcs(funcMap).Parse(`
```

Comment:
Let's keep the template name `newPR`. That also makes the diff easier to read.

Reply:
for PR marked ready to review from draft, I have kept the name `markedReadyToReviewPR`, hope thats fine